### DISCUSSION TITLE
Ogone 0.20.0

### DISCUSSION THREAD

**Discussion by SRNV** [nestdotland/nest.land#248] [Show and tell]
![ogone-svg-svg (4)](https://user-images.githubusercontent.com/37340675/89818023-5eee6a00-db49-11ea-9b77-27433b567b78.png)

**Ogone** is a web-components compiler for Deno.
to make it simple it looks like this
```typescript
<proto>
  declare:
    public subject: string = "Ogone";
</proto>
<template>Hello from ${subject}</template>
```

- [learn it](https://dev.to/srnv/deno-learn-how-to-use-ogone-486c)
- [release](https://github.com/SRNV/Ogone/releases/tag/0.20.0)
- [nest/x](https://nest.land/package/Ogone)
- [deno/x](https://deno.land/x/ogone@0.20.0)

**Comment by t8** [comment]
@SRNV awesome to hear! If you don't mind me asking, what got you interested in creating Ogone?

**Reply by SRNV** [reply]
Got some strong experiences with Vue and a little bit with React. And I wanted a tool that would complete, give some good answers to what I could experiment with Vue/React:
- Reactivity, like Richard Harris gave a way with Svelte. I mean Code reactivity not JS Reactivity (Proxies, setters and getters.. etc)
- Use of web-components which are future-proof
- Freedom from bundlers !
- Vanilla's output
- Fast compilation (actually slown down by the tsc)
- and many features that Deno can provide.

**Reply by SRNV** [reply]
Also like Deno I want all tools in one
I do think this era is prosthesis era. The goal is to have no external dependencies.
